docs/Contributing/README.md

```diff
@@ -1,25 +1,25 @@
-# Contribution
+# Contribute
 
 ### [Building Fleet](./Building-Fleet.md) 
-Provides documentation about building the code, development infrastructure, and database migrations
+Here, Fleet holds documentation about building the code, development infrastructure, and database migrations.
 
 ### [Testing](./Testing.md) 
-Includes documentation about Fleet's full test suite and integration tests
+This section includes documentation about Fleet's full test suite and integration tests.
 
 ### [Migrations](./Migrations.md)
-Information about creating and updating database migrations
+Here is information about creating and updating database migrations.
 
 ### [Committing changes](./Committing-Changes.md) 
-Contains information about how to merge changes into the codebase
+This contains information on how to merge changes into the codebase.
 
 ### [Releasing Fleet](./Releasing-Fleet.md) 
-Provides a guide for Fleet's release process
+Explore Fleet’s release process guide.
```

Comment:
```suggestion
Learn the process for how to release Fleet.
```
I'm not sure if my suggestion is any better than the original, but I don't think "explore" is the correct verb here. You "follow" a guide rather than explore it unless you want your Lego creations to be weird and wonderful.